astro/src/content/docs/apis/_user-request-body.mdx

```diff
@@ -181,7 +202,7 @@ You must specify either the **email** or the **username** or both for the User.
   </APIField>
 
   <APIField name="user.twoFactor.methods[x].email" type="String" optional>
-    The value of the email address for this method. Only present if <InlineField>user.twoFactor.methods``[x]``.method</InlineField> is `email`.
+    The value of the email address for this method. Only present if <InlineField>user.twoFactor.methods`[x]`.method</InlineField> is `email`.
```

Comment:
This was using a double backtick, and now uses a single. I saw another place in the PR where `[x]` was changed to `\[x]` with only the first bracket escaped. What were these changes trying to fix and should the fix be consistent? I still see places in the doc that format this strangely like **methods`[x]`.method**

Reply:
`eslint --fix` was applied to the entire file, including my changes (not originally this line) and hopefully fix it the right way. The status quo out there right now (see screenshot) looks weird IMO:

<img width="878" alt="image" src="https://github.com/user-attachments/assets/e6dd3311-012f-42fa-8dd7-60e68af2a2ba" />

Reply:
Your change renders the same way for me locally as the double backtick.

The `\[x]` I mentioned elsewhere (under `skipVerification`) was rendering properly, but the `\` seems unnecessary. This renders correctly with the backticks removed entirely.

It feels worth fixing even though it wasn't your change.

![image](https://github.com/user-attachments/assets/ea9e44d1-d398-4dab-aad6-4863dc28aedb)